cmake/setup.py.in

```diff
@@ -18,12 +18,12 @@ setup(
     author_email='ccppetmr@stfc.ac.uk',
     url='https://github.com/CCPPETMR/SIRF',
     platforms=['any'],
-    package_data={'sirf': ['*.so']},
+    package_data={'sirf': ['*.so', '*.dll', '*.lib']},
```

Comment:
need to add `.dylib` for OSX?

Reply:
OK, will fix `dylib`s.

> generally looks ok, but I don't know why we're doing the ugly trick to create the empty `__init__.py` with `install(CODE...)` as opposed to a simple `configure_file` as well.

the `configure_file` into an intermediate directory would still just be a blank file. Don't mind personally.